Pilot/PilotCommands.py

```diff
@@ -394,18 +402,20 @@ def __init__( self, pilotParams ):
     super( CheckWNCapabilities, self ).__init__( pilotParams )
     self.cfg = []
 
-  def execute( self ):
+  def execute( self ): #pylint: disable=R0912
```

Comment:
Please, for these entries: don't use the pylint code, but the equivalent pylint strings:

```
(DIRACGrid)  toffo@pclhcb181  ~/pyDevs/DIRAC/tests/System   v6r17-diracCertification  pylint --list-msgs | grep R0912         
:too-many-branches (R0912): *Too many branches (%s/%s)*
```

so you can put:
```
#pylint: disable=too-many-branches
```
and it will be more understandable to the reader.